### ISSUE TITLE
Unable to run the application with docker desktop.

### ISSUE THREAD

**Issue by khangnguyen2100** [thasup/microservices-ecommerce#143]
Hi just found your repo and very and I'm very impressed with your work, 
So I realy want to run your application to see and learn the technologies you use in this project. 

But I'm facing a huge problem I can't open the application even though I followed your instructions.
When I run the command "skaffold dev" everything look like works perfectly but when I access http://localhost:3000
it doesn't work.

I don't know what I did wrong and I hope you can help.
Thank you.

![Screenshot 2023-12-11 175801](https://github.com/thasup/microservices-ecommerce/assets/73072076/612b7bdb-983c-4453-8b19-6c98c0a58f71)

**Comment by thasup** [comment]
I've confirmed that it can still run without a problem on my machine.

Have you ever set your custom domain name in this file (`C:\Windows\System32\drivers\etc\hosts`) and then saved as an admin before?

```
127.0.0.1 your-custom-domain.com
```

Please also make sure that your Docker Desktop has Kubernetes enabled in its settings.

**Comment by khangnguyen2100** [comment]
Thank you very much for taking the time to help me, 

I have found my problem. because I don't have any products in the database yet so when I call API "products/bestseller" it returns me as undefined.
So there is some problem with NextJS when generate the page and it causes error. 
I tried return an empty array and it ran the project perfectly.

![image](https://github.com/thasup/microservices-ecommerce/assets/73072076/5a13beb7-5a8e-4ec0-9f10-0e4c81cb6a24)

Thank you again for your help and sharing this great project with the community. 😊😊😊
